doc/changes/devel/12473.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Changed default ECoG and sEEG electrode sizes in `mne/defaults.py` to better reflect real world sizes, by `Liberty Hamilton`_
```

Comment:
```suggestion
- Changed default ECoG and sEEG electrode sizes to better reflect real world sizes, by `Liberty Hamilton`_
```

The single ticks around `mne/defaults.py` attempt to make it an API crossref; double-backticks would work, but end users don't really need to know which file the change was in so let's just remove the file reference

Reply:
This is named according to my old PR, that's why it's failing! I'll fix the name. Thanks!